### DISCUSSION TITLE
NPM module select

### DISCUSSION THREAD

**Discussion by random777DICE** [community/community#146271] [npm]
I want to install a stable version of a specific module that is exactly compatible with my node version for react native project. How do I choose the correct version of a module?

**Comment by crazybee1001** [comment]
If you want to ensure compatibility with your React Native version, you can also use:
**npx install-peers**
This command helps install peer dependencies that are compatible with your current setup.
Thanks.

**Comment by ednamode0x22** [comment]
This is working well.
